captura/views.py

```diff
@@ -226,7 +236,7 @@ def capturista_dashboard(request):
 
 
 @login_required
-@user_passes_test(is_capturista)
+@user_passes_test(lambda u: is_member(u, [ADMINISTRADOR_GROUP, CAPTURISTA_GROUP]))
```

Comment:
This one should stay as is_capturista, based on the dicussion on today's meeting.